.github/workflows/release.yml

```diff
@@ -80,14 +80,26 @@ jobs:
         run: |
           git push origin HEAD:main
 
-      # build the package after bumping version
-      - name: Build package
+
+      - name: Build frozen and unfrozen wheels
         run: |
           poetry build
 
           # add freeze step see https://github.com/python-poetry/poetry/issues/2778
           poetry self add poetry-plugin-freeze
           poetry freeze-wheel
+          mkdir -p dist-frozen
+          cp dist/*.whl dist-frozen/
+
+          # Non-frozen wheel
+
+          poetry build -f wheel
+          for f in dist/*.whl; do
+            new_name="$(echo "$f" | sed 's/\.whl$/_unfrozen.whl/')"
```

Comment:
@mphemming I think you might have to escape the second dot before the last whl as you did in the first one. have you checked this part of the code would work in a shell?
```            new_name="$(echo "$f" | sed 's/\.whl$/_unfrozen\.whl/')"```